### DISCUSSION TITLE
Special Meaning for *.server.ts Files

### DISCUSSION THREAD

**Discussion by dresch86** [solidjs/solid-start#1437] [Q&A] [answered]
I saw on the [Session handling page](https://start.solidjs.com/advanced/session), the session code is added in a file session.server.ts. I know on Remix the *.server.ts files indicate the file should only run server-side. SolidJS uses `"use server";` but does the *.server.ts extension do the same?

**Comment by lxsmnsyc** [comment, marked as answer]
No, but you are free to use it for semantics.
